src/Form/AbstractChoiceField.php

```diff
@@ -0,0 +1,52 @@
+<?php
+
+declare(strict_types=1);
+
+namespace MyParcelCom\JsonSchema\FormBuilder\Form;
+
+use InvalidArgumentException;
+use MyParcelCom\JsonSchema\FormBuilder\Properties\SchemaProperty;
+use MyParcelCom\JsonSchema\FormBuilder\Properties\SchemaPropertyType;
+use MyParcelCom\JsonSchema\FormBuilder\Translations\LabelTranslationCollection;
+
+abstract class AbstractChoiceField implements FormElement
+{
+    protected ChoiceFieldType $fieldType;
```

Comment:
I suggest that we convert this into an abstract protected method that returns a ChoiceFieldType. The reason is that if you do that then it becomes mandatory for every child class to implement it, thus removing the overload for managing uninitialized fields. What I mean by managing uninitialized fields is that every child class has to make sure that this property has value, there is nothing to enforce it. Meanwhile, a protected abstract method basically forces you to implement it. What do you think?